.github/workflows/CI.yml

```diff
@@ -106,11 +107,16 @@ jobs:
           mkdir -p "$RAY_GEN_CACHE_DIR"
           cp -rfp . "$RAY_GEN_CACHE_DIR"
         working-directory: ${{ env.ray_dir }}
+      - name: Build ray_julia library
+        shell: julia --color=yes --project {0}
+        run: |
+          using Pkg
+          Pkg.instantiate()
+          include(joinpath(pwd(), "build_jll.jl"))
```

Comment:
Using `include("build_jll.jl")` doesn't work for some reason. We can look into that later as we need to get the CI building from source again so we can get #140 in.